recipes/ijson/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "ijson" %}
```

Comment:
It's just PRs with lots of recipes end up taking longer. Getting some of the small things out of here will help this progress.